src/constants.ts

```diff
@@ -37,7 +37,7 @@ export const CHECK_INVALID_TOKEN = process.env.CHECK_INVALID_TOKEN || 'false'
 export const VALIDATE_IN_BDRIS2 = process.env.VALIDATE_IN_BDRIS2 || 'false'
 export const CONFIRM_REGISTRATION_URL =
   process.env.CONFIRM_REGISTRATION_URL ||
-  'http://localhost:5050/confirm/registration'
+  'http://openhim-core:5001/confirm/registration'
```

Comment:
```suggestion
  'http://localhost:5001/confirm/registration'
```
I reckon we need it to be `localhost` for dev